### PR TITLE
Update README.md

### LINKED ISSUE #542: Little improvments to the doc
It's me again 😅 

I went through the installation documentation and noticed this little things : 
-> Nothing said about bundle / yarn install ; Although this project seems to be used by advanced users, I think it could be nice to remind them when to do it to avoid issues. 

-> Since Rails 5 (I think), rails supports rake commands and the norm seems nowadays to be `rails db:migrate` and `rails db:seed` instead of `rake` ; This is mainly a convention matter, not really a problem.

### PR DESCRIPTION
**Pull request by TimAle**
fix #542

### CODE REVIEW

**Review by GideonEXE — APPROVED** [review]

**Review by Vin0uz — COMMENTED** [review]
Why is there `1.` everywhere? 🙈

**Comment by TimAle** [comment]
> Why is there `1.` everywhere? 🙈

It's just how markdown works :)
https://www.markdownguide.org/basic-syntax/#ordered-lists

File: README.md [hunk 1]
```diff
@@ -69,21 +69,26 @@ You can login to [Camunda Admin interface](http://localhost:8766/camunda) with c
 
     * [Without Oracle Instant Client](https://github.com/latera/homs/blob/master/WITHOUT_ORACLE.md).
 
-2. Start `webpack-dev-server` in source directory:
+1. Install [Yarn](https://github.com/yarnpkg/yarn#installing-yarn) and run
+   ```
+   yarn install
+   ```
+
+1. Start `webpack-dev-server` in source directory:
```
Comment (GideonEXE) [inline README.md:77]: 2 i think

**Review by GideonEXE — CHANGES_REQUESTED** [review]

**Review by GideonEXE — APPROVED** [review]

**Comment by Vin0uz** [comment]
> > Why is there `1.` everywhere? 🙈
> 
> It's just how markdown works :)
> https://www.markdownguide.org/basic-syntax/#ordered-lists

😮  I always thought numbers mattered, thanks for that !

**Pull request merged**